### PR TITLE
AssetTimeseriesPanel styling and documentation

### PR DESCRIPTION
**Pull request by joyrostie**
AssetTimeseriesPanel styling and documentation
AssetDocumentsPanel storybook documentation
AssetEventsPanel storybook documentation
AssetMeta storybook documentation

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-306.gearboxjs.preview.cogniteapp.com

File: src/components/AssetDocumentsPanel/stories/full.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ function ExampleComponent(props) {
 | `documentRenderer`    | Custom render functions for documents                                  | `string`                                      |`DocumentRenderer` |
 | `customCategorySort`  | Sort function used to sort categories after priority categories        | `(a: string, b: string) => number`            |             |
 | `customSpinner`       | A custom spinner to be shown in tabs while data is being loaded        | `React.ReactNode`                             |             |
-| `styles`              | Object that defines inline CSS styles inner elements of the component  | `DocumentTableStyles`                         |             |
+| `styles`              | Object that defines inline CSS styles inner elements of the component  | `AssetDocumentsPanelStyles`                   |             |
```
Comment (xitman78) [inline src/components/AssetDocumentsPanel/stories/full.md]:
```suggestion
| `styles`              | Object that defines inline CSS styles for inner elements of the component  | `AssetDocumentsPanelStyles`                   |             |
```

File: src/components/AssetTimeseriesPanel/stories/AssetTimeseriesPanel.stories.tsx [hunk 2]
```diff
@@ -83,4 +85,20 @@ storiesOf('AssetTimeseriesPanel/Examples', module)
         content: customTimeseriesChartMeta,
       },
     }
+  )
+  .add(
+    'With custom stylrs',
```
Comment (xitman78) [inline src/components/AssetTimeseriesPanel/stories/AssetTimeseriesPanel.stories.tsx]:
```suggestion
    'With custom styles',
```

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-306.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-306.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-306.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-306.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-306.gearboxjs.preview.cogniteapp.com

**Pull request merged**